api/RTCPeerConnection.json

```diff
@@ -1878,6 +1880,56 @@
           }
         }
       },
+      "icecandidateerror_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/icecandidateerror_event",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#event-icecandidateerror",
+          "description": "<code>icecandidateerror</code> event",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "14.1"
```

Comment:
Seems correct, https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=270101 points to this version too, and this matches existing data.